terragrunt/aws/cloudfront/cloudfront.tf

```diff
@@ -91,7 +91,7 @@ resource "aws_cloudfront_response_headers_policy" "security_headers_api" {
       override = true
     }
     content_security_policy {
-      content_security_policy = "report-uri https://csp-report-to.security.cdssandbox.xyz/report; default-src 'none'; script-src 'self'; connect-src 'self'; img-src 'self'; style-src 'self'; frame-ancestors 'self'; form-action 'self';"
+      content_security_policy = "report-uri https://csp-report-to.security.cdssandbox.xyz/report; default-src 'none'; script-src 'self' 'unsafe-inline' https://stackpath.bootstrapcdn.com/bootstrap/ https://code.jquery.com https://ikuhdso72yomvbpvnwxy4odtna0fxnec.lambda-url.ca-central-1.on.aws/static/js/ https://cdnjs.cloudflare.com/ajax/libs/ https://cdnjs.cloudflare.com/ajax/libs/; font-src 'self' https://fonts.gstatic.com; connect-src 'self'; img-src 'self' https://ikuhdso72yomvbpvnwxy4odtna0fxnec.lambda-url.ca-central-1.on.aws/static/img/ http://www.w3.org/2000; style-src 'self' 'unsafe-inline' https://ikuhdso72yomvbpvnwxy4odtna0fxnec.lambda-url.ca-central-1.on.aws/static/css/  https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css https://fonts.gstatic.com https://fonts.googleapis.com; frame-ancestors 'self'; form-action 'self';"
```

Comment:
My concern here would be that we are hardcoding a dynamic URL for a specific lambda. Ideally we would pull that value out of a variable (https://github.com/cds-snc/url-shortener/blob/main/terragrunt/env/staging/env_vars.hcl#L5) and then also either have relative paths for those assets, or deploy the application with the URL pre-populated from the ENV variable here (https://github.com/cds-snc/url-shortener/blob/main/terragrunt/aws/api/lambda.tf#L18)

Reply:
Good point Max! I will make the change.